### PR TITLE
feat: the sign up screen sends the registration data to api

### PR DESCRIPTION
**Pull request by PauloCruz06**

### CODE REVIEW

File: src/components/Signup.js [hunk 1]
```diff
@@ -11,51 +11,122 @@ export default function Signup(){
...
+
+        promise.then(() => {
+            navigate("/");
+        });
+        promise.catch(() => {
+            alert("Erro em fazer cadastro");
+            setLoading(false);
+        });
+
+        return (
```
Comment (SauloDriven) [inline src/components/Signup.js]: there is a lot of repeated code in that component, this can be changed with a little better logic.

**Review by SauloDriven — CHANGES_REQUESTED** [review]
As long as I have this code all repeated I can't accept this PR :cry:

File: src/components/Signup.js (hunk 1 above)
Comment (PauloCruz06) [reply src/components/Signup.js]: Ok :(

File: src/components/Signup.js [hunk 2]
```diff
@@ -33,101 +33,63 @@ export default function Signup(){
             body
         );
 
-        promise.then(() => {
+        promise.then((re) => {
+            console.log(re.data);
```
Comment (SauloDriven) [inline src/components/Signup.js]: console never :cry:

Comment (PauloCruz06) [reply src/components/Signup.js]: ahh, sorry man

**Review by SauloDriven — CHANGES_REQUESTED** [review]
Nice Nice, remove console man pleaseee :frog:

**Review by SauloDriven — APPROVED** [review]
Thanks

**Pull request merged**